playground/backend/internal/setup_tools/life_cycle/life_cycle_setuper.go

```diff
@@ -58,3 +75,70 @@ func Setup(sdk pb.Sdk, code string, pipelineId uuid.UUID, workingDir string, pre
 	}
 	return lc, nil
 }
+
+// prepareGoFiles prepares file for Go environment.
+// Copy go.mod and go.sum file from /path/to/preparedModDir to /path/to/workingDir.
+func prepareGoFiles(lc *fs_tool.LifeCycle, preparedModDir, workingDir string, pipelineId uuid.UUID) error {
+	if err := lc.CopyFile(goModFileName, preparedModDir, workingDir); err != nil {
+		logger.Errorf("%s: error during copying %s file: %s\n", pipelineId, goModFileName, err.Error())
+		return err
+	}
+	if err := lc.CopyFile(goSumFileName, preparedModDir, workingDir); err != nil {
+		logger.Errorf("%s: error during copying %s file: %s\n", pipelineId, goSumFileName, err.Error())
+		return err
+	}
+	return nil
+}
+
+// prepareJavaFiles prepares file for Java environment.
+// Copy log config file from /path/to/preparedModDir to /path/to/workingDir/executable_files/{pipelineId}
+//	and update this file according to pipeline.
+func prepareJavaFiles(lc *fs_tool.LifeCycle, workingDir string, pipelineId uuid.UUID) error {
+	err := lc.CopyFile(javaLogConfigFileName, workingDir, lc.Folder.BaseFolder)
+	if err != nil {
+		logger.Errorf("%s: error during copying logging.properties file: %s\n", pipelineId, err.Error())
+		return err
+	}
+	err = updateLogConfigFile(lc)
+	if err != nil {
+		logger.Errorf("%s: error during updating logging.properties file: %s\n", pipelineId, err.Error())
+		return err
+	}
+	return nil
+}
+
+// updateLogConfigFile updates log config file according to pipeline
+func updateLogConfigFile(lc *fs_tool.LifeCycle) error {
```

Comment:
Is it only for Java? If yes, let's rename it accordingly like `updateJavaLogConfigFile`

Reply:
Changed.